### PR TITLE
add divide method dasks series and dataframe

### PR DESCRIPTION
**Pull request by msbrown47**
add divide method dasks series and dataframe related to #1259

- [ ] Tests added / passed
- [ ] Passes `black dask` / `flake8 dask`

### CODE REVIEW

File: dask/dataframe/tests/test_arithmetics_reduction.py [hunk 1]
```diff
@@ -641,6 +643,7 @@ def test_frame_series_arithmetic_methods():
             assert_eq(l.sub(r, axis=axis), el.sub(er, axis=axis))
             assert_eq(l.mul(r, axis=axis), el.mul(er, axis=axis))
             assert_eq(l.div(r, axis=axis), el.div(er, axis=axis))
+            assert_eq(l.divide(r, axis=0), el.divide(er, axis=0))
```
Comment (jrbourbeau) [inline dask/dataframe/tests/test_arithmetics_reduction.py]: This part of the test is checking that multiple values for `axis=` are properly supported. Updating this line to `axis=axis` should ensure that things `divide` is working as expected

```suggestion
            assert_eq(l.divide(r, axis=axis), el.divide(er, axis=axis))
```

File: dask/dataframe/tests/test_arithmetics_reduction.py [hunk 2]
```diff
@@ -588,6 +588,7 @@ def test_frame_series_arithmetic_methods():
             # pandas-26793
             warnings.simplefilter("ignore", RuntimeWarning)
             assert_eq(l.div(r, fill_value=0), el.div(er, fill_value=0))
+            assert_eq(l.divide(r, axis=0), el.divide(er, axis=0))
```
Comment (jrbourbeau) [inline dask/dataframe/tests/test_arithmetics_reduction.py]: This section of the test is testing the `fill_value` parameter

```suggestion
            assert_eq(l.divide(r, fill_value=0), el.divide(er, fill_value=0))
```

**Comment by TomAugspurger** [comment]
Looks good. I think merging master & re-pushing should fix the CI failure.

**Comment by jrbourbeau** [comment]
Thanks for keeping on top of this @TomAugspurger. Will merge on green.

**Comment by jrbourbeau** [comment]
Thanks @msbrown47!

Also, I noticed this was your first code contribution to this repository. Welcome! Hope to see you at SciPy again next year : )

**Pull request merged**